src/Exceptions/HttpRangeNotSatisfiable.php

```diff
@@ -8,4 +8,3 @@ class HttpRangeNotSatisfiable extends HttpException {
     public const DESCRIPTION = "Indicates that none of the ranges in the request's Range header field overlap the current extent of the selected resource or that the set of ranges requested has been rejected due to invalid ranges or an excessive request of small or overlapping ranges.";
     public const HREF = "https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/416";
 }
-
```

Comment:
Removed unix line ending - ALWAYS have a unix line ending, it can cause very weird issues.

Just use https://marketplace.visualstudio.com/items?itemName=jkiviluoto.tws you can use this to add them automatically. It also removes trailing whitespaces